lib/User_Proxy.php

```diff
@@ -32,9 +32,20 @@
 use OCP\User\IProvidesEMailBackend;
 use OCP\User\IProvidesExtendedSearchBackend;
 use OCP\User\IProvidesQuotaBackend;
+use OCP\User\IProvidesUserNameBackend;
 use OCP\UserInterface;
 
-class User_Proxy extends Proxy implements IUserBackend, UserInterface, IProvidesQuotaBackend, IProvidesExtendedSearchBackend, IProvidesEMailBackend {
+class User_Proxy
+	extends
+		Proxy
+	implements
+		IUserBackend,
+		UserInterface,
+		IProvidesQuotaBackend,
+		IProvidesExtendedSearchBackend,
+		IProvidesEMailBackend,
+		IProvidesUserNameBackend
```

Comment:
since this only exists in 10.0.10 through https://github.com/owncloud/core/pull/32579/, we need to set min-version to 10.0.10

Reply:
@butonic please adjust